spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTypeConverter.java

```diff
@@ -38,6 +50,17 @@ void init() {
     gravitinoToSparkTypeMapper.put(StringType.get(), DataTypes.StringType);
```

Comment:
could you reorganize the sequence like SparkTypeConverter?

Reply:
Fixed.